build/package/generate-msi.sh

```diff
@@ -23,32 +23,31 @@ export NOTARY_SIGNING_KEY
 
 go-msi check-env
 
-VERSION="$(git tag --list "${TOOL_NAME:?}/v*" --sort=taggerdate | tail -1 | cut -d "v" -f 2)"
-
+VERSION_GIT="$(git describe --match "${TOOL_NAME:?}/v*" | cut -d "v" -f 2)"
```

Comment:
bringing back a version that includes the state of the branch so uploaded nuspec can be distinguished from production releases